front_end/entrypoint_template.html

```diff
@@ -14,7 +14,7 @@
     }
   }
 </style>
-<meta http-equiv="Content-Security-Policy" content="object-src 'none'; script-src 'self' 'unsafe-eval' https://chrome-devtools-frontend.appspot.com">
+<meta http-equiv="Content-Security-Policy" content="worker-src 'self' blob: ; object-src 'none'; script-src 'self' 'unsafe-eval' https://chrome-devtools-frontend.appspot.com">
```

Comment:
I wish this was more self-documenting. Might be worth an explanatory comment as part of the file.

Reply:
I thought about moving this to `build_entrypoints` script, so only `rn_fusebox.html` entrypoint gets this.

Will add this tomorrow and add an explanatory comment, basically RDT frontend is using workers API for file uploads.